data-plane/config/broker/100-config-kafka-broker-data-plane.yaml

```diff
@@ -40,7 +40,7 @@ data:
     receive.buffer.bytes=-1
     request.timeout.ms=30000
     enable.idempotence=false
-    # interceptor.classes=""
+    interceptor.classes="io.cloudevents.kafka.PartitionKeyExtensionInterceptor"
```

Comment:
Maybe it's better to hardcode this? Or at least make users aware that changing this will break partition key extension?

Reply:
Done!